statsd/client.py

```diff
@@ -146,7 +146,7 @@ def _send(self, data):
         """Send data to statsd."""
         try:
             self._sock.sendto(data.encode('ascii'), self._addr)
-        except (socket.error, RuntimeError):
+        except (socket.error, socket.gaierror, RuntimeError):
```

Comment:
I don't believe this does what you think it does. `socket.gaierror` is a subclass of `socket.error`, so should have already been caught, if it were thrown from here:

```py
try:
    raise socket.gaierror()
except socket.error as e:
    print('hi there')
```